src/main/features/response/draft/responseDraft.ts

```diff
@@ -81,4 +81,18 @@ export class ResponseDraft extends DraftDocument implements Serializable<Respons
       && this.rejectPartOfClaim !== undefined
       && this.rejectPartOfClaim.option === RejectPartOfClaimOption.PAID_WHAT_BELIEVED_WAS_OWED
   }
+
+  public requireMediation (): boolean {
```

Comment:
Also `public` is the default modifier in TypeScript, can be removed.

Reply:
Thank you, i didn't know that. However the rest of the methods include the public modifier, so i'll keep it in for consistency.

Reply:
@rlewan I'd rather leave `public` even though it's public by default. (In Java it's package accessible by default and probably in other languages is different). 

I know it might be redundant, but I don't mind this kind of redundancy.